deploy_webserver.cmd

```diff
@@ -1,2 +0,0 @@
-cd web
-npm install .
```

Comment:
Why did you delete this, my friend ? It should have been renamed like you did with the other script files.

Reply:
This is just git's behavior. The squashed commit thinks it's another files, but in commit history it's not.

Reply:
![Ok](https://memegenerator.net/img/instances/43292462/you-won-this-time-but-i-will-have-my-revenge.jpg "You won this time")